[BOJ]1753.py

```diff
@@ -0,0 +1,44 @@
+# 백준 - https://www.acmicpc.net/problem/1753
+
+import sys
+import math
+import heapq
+
+INF = math.inf
+
+
+def dijkstra(start):
+    queue = []
+    heapq.heappush(queue, (0, start))
+    distance[start] = 0
+
+    while queue:
+        dist, now = heapq.heappop(queue)
+        if distance[now] < dist:
+            continue
+
+        for vertex, weight in graph[now]:
+            cost = dist + weight
+
+            if cost < distance[vertex]:
```

Comment:
여기서 바다코끼리를 써서 풀면 더 좋겠네요. 아래에서 계속 사용하고 있으니까요 ^^
```suggestion
            if (cost := dist + weight) < distance[vertex]:
```